proto/cosmos/crisis/v1beta1/tx.proto

```diff
@@ -14,3 +20,6 @@ message MsgVerifyInvariant {
   string invariant_module_name = 2 [(gogoproto.moretags) = "yaml:\"invariant_module_name\""];
   string invariant_route       = 3 [(gogoproto.moretags) = "yaml:\"invariant_route\""];
 }
+
+// MsgVerifyInvariantResponse defines the Msg/VerifyInvariant response type.
+message MsgVerifyInvariantResponse {  }
```

Comment:
same as https://github.com/cosmos/cosmos-sdk/pull/7557/files#r505568809

Reply:
let's keep as is because of style conventions